api/v1/controllers/samples.js

```diff
@@ -57,21 +93,33 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   findSamples(req, res, next) {
+    // Check if Cache is on for Wildcard Sample query
+    if (featureToggles.isFeatureEnabled('cacheGetSamplesWildcard')) {
+      const query = req.query.name;
+      helper.cacheEnabled =
+        query ? (query.indexOf('*') > -1) ? true : false : false;
+      helper.cacheKey =  helper.cacheEnabled ? query : null;
+      helper.cacheExpiry = helper.cacheEnabled ?
+        parseInt(getSamplesWildcardCacheInvalidation) : null;
```

Comment:
Is specifying the radix required in parseInt?

Reply:
Yes time in second.